Update migrating-to-github-container-registry-for-docker-images.md (#5886)
github/docs: content/packages/working-with-a-github-packages-registry/migrating-to-the-container-registry-from-the-docker-registry.md

2. In your {% data variables.product.prodname_actions %} workflow file, update the package url from `https://docker.pkg.github.com` to `ghcr.io`.

> 3. Use the `GITHUB_TOKEN` for your authentication personal access token (PAT) as a GitHub Actions secret. {% data variables.product.prodname_container_registry %} now supports using `GITHUB_TOKEN` for your PAT. For more information, see "[Authenticating with the Github Token](/actions/configuring-and-managing-workflows/authenticating-with-the-github_token)."

Reviewer: I think we could shorten this step to just:
```suggestion
3. Use the `GITHUB_TOKEN` for your authentication personal access token (PAT). For more information, see "[Authentication in a workflow](/actions/reference/authentication-in-a-workflow)."
```

> 3. Use the `GITHUB_TOKEN` for your authentication personal access token (PAT) as a GitHub Actions secret. {% data variables.product.prodname_container_registry %} now supports using `GITHUB_TOKEN` for your PAT. For more information, see "[Authenticating with the Github Token](/actions/configuring-and-managing-workflows/authenticating-with-the-github_token)."

Reviewer: I think we could shorten this step to just:
```suggestion
3. Use the `GITHUB_TOKEN` for your authentication personal access token (PAT). For more information, see "[Authentication in a workflow](/actions/reference/authentication-in-a-workflow)."
```

Author: @hubwriter  Hi, I just did that!
